apps/cowswap-frontend/src/modules/swap/state/slippageValueAndTypeAtom.ts

```diff
@@ -0,0 +1,72 @@
+import { atom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
+
+import { DEFAULT_SLIPPAGE_BPS, MINIMUM_ETH_FLOW_SLIPPAGE_BPS } from '@cowprotocol/common-const'
+import { bpsToPercent } from '@cowprotocol/common-utils'
+import { mapSupportedNetworks, SupportedChainId } from '@cowprotocol/cow-sdk'
+import { walletInfoAtom } from '@cowprotocol/wallet'
+
+import { isEoaEthFlowAtom } from './isEoaEthFlowAtom'
+
+type SlippageBpsPerNetwork = Record<SupportedChainId, number | null>
+
+type SlippageType = 'smart' | 'default' | 'user'
+
+const normalSwapSlippageAtom = atomWithStorage<SlippageBpsPerNetwork>(
+  'swap-slippage-atom:v0',
+  mapSupportedNetworks(null)
+)
+
+const ethFlowSlippageAtom = atomWithStorage<SlippageBpsPerNetwork>(
+  'eth-flow-slippage-atom:v0',
```

Comment:
Sorry again, one more thing I noticed only now.

We use a mix of this and camelCase for localStorage keys.

Checking on the source, `6` are in this pattern, and `9` are using camelCase.

https://github.com/search?q=repo%3Acowprotocol%2Fcowswap+%22atom%3Av%22&type=code

I rather have a single format.
Since camelCase is the majority, let's go with that and slowly move the others to it as well.

Reply:
Fixed